unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/KernelCommunication/WebSocketCommunication/WebSocketCommunication.cs

```diff
@@ -26,37 +26,66 @@ public class WebSocketCommunication : IKernelCommunication
 
     private string StartServer(int port, int maxPort)
     {
+        var debugConfig = GameObject.Find("DebugConfig").GetComponent<DebugConfigComponent>();
```

Comment:
i would try to find another way of getting DebugConfigComponent because finding objects by its name could lead into errors. You could have any other object called the same way in the scene or someone could change the name of it easily. It would be better to inject the component into WebSocketCommunication constructor. DebugConfigComponent could have its own singleton or a shared instance somewhere.